### PR TITLE
Fix user account set up transaction

### PR DESCRIPTION
**Pull request by judezhu**

### CODE REVIEW

File: transactions/scripts/users/is_account_all_set_up.cdc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    let account = getAccount(address)
+    return account.getCapability<&{
+            NonFungibleToken.Receiver,
+            NonFungibleToken.CollectionPublic,
+            TopShot.MomentCollectionPublic,
+            MetadataViews.ResolverCollection
+        }>(/public/MomentCollection).check() &&
+        account.getCapability<&{
+            NonFungibleToken.CollectionPublic
+        }>(PackNFT.CollectionPublicPath).check()
```
Comment (Deewai) [inline transactions/scripts/users/is_account_all_set_up.cdc:17]: this should be the moment collection. something like
```suggestion
        }>(/storage/MomentCollection).check()
```

File: transactions/scripts/users/is_account_all_set_up.cdc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import TopShot from 0xTOPSHOTADDRESS
+import NonFungibleToken from 0xNFTADDRESS
+import PackNFT from 0xPACKNFTADDRESS
```
Comment (Deewai) [inline transactions/scripts/users/is_account_all_set_up.cdc:3]: we don't need the PackNFT import

File: transactions/user/setup_up_all_collections.cdc [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import NonFungibleToken from 0xNFTADDRESS
+import TopShot from 0xTOPSHOTADDRESS
+import MetadataViews from 0xMETADATAVIEWSADDRESS
```
Comment (Deewai) [inline transactions/user/setup_up_all_collections.cdc:3]: we need to import PackNFT collection

File: transactions/scripts/users/is_account_all_set_up.cdc (hunk 1 above)
Comment (Deewai) [reply transactions/scripts/users/is_account_all_set_up.cdc:17]: ah, you are correct

File: transactions/scripts/users/is_account_all_set_up.cdc (hunk 2 above)
Comment (Deewai) [reply transactions/scripts/users/is_account_all_set_up.cdc:3]: ignore this

File: transactions/user/setup_up_all_collections.cdc (hunk 3 above)
Comment (judezhu) [reply transactions/user/setup_up_all_collections.cdc:3]: added

**Review by Deewai — APPROVED** [review]

**Comment by tpetrychyn** [comment]
@judezhu can you remind me how/where these tx are used?

**Review by tpetrychyn — APPROVED** [review]

**Pull request merged**